db/c_test.c

```diff
@@ -2,12 +2,13 @@
    Use of this source code is governed by a BSD-style license that can be
    found in the LICENSE file. See the AUTHORS file for names of contributors. */
 
+#include <stdio.h>
```

Comment:
Since it is just to fix a test, I'm going to merge it. But I don't get how this can fix the problem.